drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/main/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/rightpanel/TestRunnerReportingPanelWrapperImpl.java

```diff
@@ -45,6 +48,10 @@ public void onTestRun(TestResultMessage testResultMessage) {
 
     @Override
     public IsWidget asWidget() {
-        return testRunnerReportingPanel.asWidget();
+        /* Here, a FlowPanel is added to align the padding with other DocksPanels */
+        FlowPanel flowPanel = GWT.create(FlowPanel.class);
+        flowPanel.getElement().getStyle().setPaddingTop(4, Style.Unit.PX);
+        flowPanel.add(testRunnerReportingPanel.asWidget());
+        return flowPanel;
```

Comment:
Here, I created a new  `FlowPanel` which wraps `TestRunnerReportingPanel` widget adding the missing padding pixels. Consider that isn't possible to modify the original panel, nor extends it to override its style.